src/v2/cookbook/packaging-sfc-for-npm.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: Packaging Vue Components for npm
+title: Paquete de Componentes de Vue para npm
 type: cookbook
 order: 12
 ---
 
 ## Base Example
 
-Vue components by nature are meant to be re-used. This is easy when the component is only used within a single application. But how can you write a component once and use it in multiple sites/applications? Perhaps the easiest solution is via npm.
+Los componentes de Vue están destinados a ser reusados. Es sencillo cuando el componente solo es usado dentro de una aplicación sencilla. ¿Pero como se escribe un componente solo una vez y es usado en múltiples sitios/aplicaciones? Quizá la solución más fácil es via npm.
```

Comment:
- Please change this sentence `¿Pero como se escribe un componente solo una vez y es usado en múltiples sitios/aplicaciones?` to `Pero, ¿cómo puede escribir un componente una vez y usarlo en múltiples sitios/aplicaciones?`

Reply:
Hola Seba voy a seguir tu sugerencia pero no se puede comenzar una oración entre signos de interrogación comenzando con minúscula, lo que haré es incluir el Pero dentro también.

Reply:
Yes, you're right. Please change according to your comment. Thanks!